tests/test_modeling_reformer.py

```diff
@@ -0,0 +1,1123 @@
+        }
+        return config
+
+    def _get_hidden_states(self):
+        return torch.tensor(
+            [
+                [
+                    [
+                        1.90826353e00,
+                        -1.45999730e00,
+                        -6.20405462e-01,
+                        1.52503433e00,
+                        -3.64464232e-01,
+                        -8.27359235e-01,
+                        8.39670803e-01,
+                        2.44492178e-01,
+                        4.98332758e-01,
+                        2.69175139e00,
+                        -7.08081422e-03,
+                        1.04915401e00,
+                        -1.83476661e00,
+                        7.67220476e-01,
+                        2.98580543e-01,
+                        2.84803992e-02,
+                    ],
+                    [
+                        -2.66374286e-02,
+                        4.33497576e-01,
+                        3.10386309e-01,
+                        5.46039944e-01,
+                        -2.47292666e-04,
+                        -7.52305019e-01,
+                        2.39162103e-01,
+                        7.25216186e-01,
+                        -7.58357372e-01,
+                        4.20635998e-01,
+                        -4.04739919e-02,
+                        1.59924145e-01,
+                        2.05135748e00,
+                        -1.15997978e00,
+                        5.37166397e-01,
+                        2.62873606e-01,
+                    ],
+                    [
+                        1.85247482e-01,
+                        7.07046037e-01,
+                        -6.77089715e-01,
+                        -2.24209655e00,
+                        -3.75307980e-02,
+                        -8.59380874e-01,
+                        -2.81027884e00,
+                        1.01276376e00,
+                        -1.69438001e00,
+                        4.17574660e-01,
+                        -1.49196962e00,
+                        -1.76483717e00,
+                        -1.94566312e-01,
+                        -1.71183858e00,
+                        7.72903565e-01,
+                        -1.11557056e00,
+                    ],
+                    [
+                        9.46069193e-01,
+                        1.53417623e-01,
+                        -9.58686996e-01,
+                        1.18126669e-01,
+                        1.75967724e00,
+                        1.62194590e00,
+                        -5.74108159e-01,
+                        6.79920443e-01,
+                        5.44028163e-01,
+                        2.05466114e-01,
+                        -3.63045868e-01,
+                        2.41865062e-01,
+                        3.20348382e-01,
+                        -9.05611176e-01,
+                        -1.92690727e-01,
+                        -1.19917547e00,
+                    ],
+                ]
+            ],
```

Comment:
comment where these are from? Trax code? or your code to ensure no drift?

Reply:
Yeah, that's written right under the `ReformerIntegrationTests class`. Kinda hard to see in this PR since it's just a single line.